_posts/2023-07-12-future-career-growth-for-frontend-engineer.md

```diff
@@ -0,0 +1,180 @@
+---
+title: "Future career growth for Frontend Engineer: aspects which you need to know besides frontend"
+excerpt: "Self-development tips for front-end developers"
```

Comment:
Hi @Algoritm211 - shall we stick to using `frontend` at all places as currently I see a mix of `frontend` and `front-end`?

Reply:
Thank you very much for the comment, fixed, in all places `frontend` now